addons/website_slides/static/tests/tours/slide_course_publisher_standard.js

```diff
@@ -63,7 +63,8 @@ tour.register('course_publisher_standard', {
     content: 'eLearning: is the Corgi set ?',
     trigger: 'img.o_wslides_course_pict',
     run: function () {
-        if ($('img.o_wslides_course_pict').attr('src').endsWith('GoldWinnerPembrookeWelshCorgi.jpg')) {
+        let regex=/GoldWinnerPembrookeWelshCorgi.jpg/;
```

Comment:
Two things here: if you don't plan to change the regex, better use `const`;
It's also better to use a more indicative name, like below. 
```suggestion
        const expectedImageUrlRegex=/GoldWinnerPembrookeWelshCorgi.jpg/;
```